db/migrate/20180125155315_add_details_to_users.rb

```diff
@@ -0,0 +1,10 @@
+class AddDetailsToUsers < ActiveRecord::Migration[5.1]
+  def change
+    change_table(:users) do |t|
+      t.column :date_of_birth, :datetime
+      t.string :avatar, default: 'no-avatar.png'
```

Comment:
this is incorrect way how to show default images for carrierwave. check how with page or company it works